src/courses/mappers/14.md

```diff
@@ -0,0 +1,1697 @@
+---
+order: 14
+next: 15.md
+title: Implementing a Mapper
+author: Charles Hu
+---
+
+## Mapper Introduction
+
+::: note OHDF Converters Utilities
+Special utilities from OHDF Converters are used here. [Refer here](./09.md) for a refresher on OHDF Converters.
+:::
+
+Now that we have developed a mapping, we can implement that mapping as a mapper which applies the mapping to any compatible file for a conversion to OHDF.
+
+## File Set Up
+
+A number of crucial files that support and provide the infrastructure needed for the *-to-OHDF mapper have to be set up first before we begin actual mapper development.
+
+::: note Specialized Security Tools
```

Comment:
It may be a little much, but I want to explicitly call out points where the end result created by a user may differ greatly based on the type of data they're using.